ConsumerApi.Tests.Integration/StepDefinitions/IdentitiesApiStepDefinitions.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Backbone.ConsumerApi.Tests.Integration.API;
+using Backbone.ConsumerApi.Tests.Integration.Configuration;
+using Backbone.ConsumerApi.Tests.Integration.Extensions;
+using Backbone.ConsumerApi.Tests.Integration.Models;
+using Backbone.Crypto;
+using Backbone.Crypto.Implementations;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace Backbone.ConsumerApi.Tests.Integration.StepDefinitions;
+
+[Binding]
+[Scope(Feature = "POST Identity")]
+internal class IdentitiesApiStepDefinitions : BaseStepDefinitions
+{
+    private readonly ChallengesApi _challengeApi;
+    private readonly IdentitiesApi _identitiesApi;
+    private HttpResponse<CreateIdentityResponse>? _identityResponse;
+    private HttpResponse<Challenge>? _challengeResponse;
+
+    public IdentitiesApiStepDefinitions(IOptions<HttpConfiguration> httpConfiguration, IdentitiesApi identitiesApi, ChallengesApi challengeApi) : base(httpConfiguration)
+    {
+        _identitiesApi = identitiesApi;
+        _challengeApi = challengeApi;
+    }
+
+    [Given(@"a Challenge c")]
+    public async Task GivenAChallengeC()
+    {
+        _challengeResponse = await _challengeApi.CreateChallenge(_requestConfiguration);
+        _challengeResponse!.IsSuccessStatusCode.Should().BeTrue();
+    }
+
+    [When(@"a POST request is sent to the /Identities endpoint")]
+    public async Task WhenAPOSTRequestIsSentToTheIdentitiesEndpoint()
+    {
+        var serializedChallenge = JsonConvert.SerializeObject(new
+        {
+            _challengeResponse!.Content.Result!.Id,
+            _challengeResponse!.Content.Result!.ExpiresAt,
+            Type = "Identity"
+        });
```

Comment:
You should be able to just serialize `_challengeResponse!.Content.Result!` instead of all this.